### ISSUE TITLE
Linux Mint 20.1: Runs into endless loop

### ISSUE THREAD

**Issue by Hornochs** [saveriomiroddi/zfs-installer#195]
Hey there,
so I had an stupid issue in Kubuntu which prevents me to use three of my screens. Therefore I wanted to switch to Linux Mint Cinnamon 20.1. Well, installation with your script doesn't work now :(

Starting the script, gives me the welcome screen what will be done. Well and then this appears in the Terminal:

```
mint@mint:~$ GET https://git.io/JelI5 | sudo bash

###############################################################################
# activate_debug
###############################################################################

###############################################################################
# set_distribution_data
###############################################################################

###############################################################################
# store_os_distro_information
###############################################################################
No LSB modules are available.

###############################################################################
# store_running_processes
###############################################################################

###############################################################################
# check_prerequisites
###############################################################################

###############################################################################
# display_intro_banner
###############################################################################

###############################################################################
# check_system_memory
###############################################################################

###############################################################################
# save_disks_log
###############################################################################

###############################################################################
# find_suitable_disks
###############################################################################
v_suitable_disks: "/dev/disk/by-id/ata-Samsung_SSD_860_EVO_1TB_S4X6NJ0N423034W" "/dev/disk/by-id/nvme-Samsung_SSD_970_EVO_Plus_1TB_S4EWNM0R208642B" "/dev/disk/by-id/nvme-eui.0025385211902abc"


###############################################################################
# register_exit_hook
###############################################################################

###############################################################################
# create_passphrase_named_pipe
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################
...
...
...

###############################################################################
# prepare_standard_repositories_Linuxmint
###############################################################################

^C
Currently set exports, for performing an unattended (as possible) installation with the same configuration:

export ZFS_USE_PPA=
export ZFS_SELECTED_DISKS=
export ZFS_BOOT_PARTITION_SIZE=
export ZFS_PASSPHRASE=''
export ZFS_DEBIAN_ROOT_PASSWORD=''
export ZFS_RPOOL_NAME=
export ZFS_BPOOL_CREATE_OPTIONS=""
export ZFS_RPOOL_CREATE_OPTIONS=""
export ZFS_POOLS_RAID_TYPE=
export ZFS_NO_INFO_MESSAGES=1
export ZFS_SWAP_SIZE=
export ZFS_FREE_TAIL_SPACE=

```

Like really endless. So I pressed "Control-C" :) 

Log files are attached.
[disks.log](https://github.com/saveriomiroddi/zfs-installer/files/6406689/disks.log)
[install.log](https://github.com/saveriomiroddi/zfs-installer/files/6406690/install.log)
[os_information.log](https://github.com/saveriomiroddi/zfs-installer/files/6406691/os_information.log)
[running_processes.log](https://github.com/saveriomiroddi/zfs-installer/files/6406692/running_processes.log)

Any Idea?

**Comment by Hornochs** [comment]
Did some debugging, looks like the second Commands run into an issue:

```
mint@mint:~$ sudo perl -i -pe 's/^(deb cdrom)/# $1/' /etc/apt/sources.list
mint@mint:~$ sudo add-apt-repository --yes --no-update universe
Usage: add-apt-repository [options] repository

add-apt-repository: error: no such option: --no-update
```

**Comment by saveriomiroddi** [comment]
Hello!

Thanks for the report. Sorry for that, but until I have debootstrap scripts for all the distributions (assuming all the distros can be deboostrapped), I can't realistically test all the small differences :cursing_face:.

Having said that, I'm fixing this shortly.

**Comment by Hornochs** [comment]
Heyho,
No worries! If I was 100% sure I would put up an pull request and fixed it on my own, since the installation went fine after that. Maybe I get an Idea about that.

**Comment by saveriomiroddi** [comment]
The fix for the `--update` is technically simple, but requires a bit of thought in order to be implemented very cleanly. Will close at the end of today :)

**Comment by saveriomiroddi** [comment]
I'll also split the issue, in order to track it better.

**Comment by saveriomiroddi** [comment]
@Hornochs Thanks for the bug-hunting help :) I've closed all the issues, and released new versions.
